festival/src/main/java/com/hubses/festival/service/NewsService.java

```diff
@@ -0,0 +1,72 @@
+package com.hubses.festival.service;
+
+import com.hubses.festival.domain.News;
+import com.hubses.festival.dto.form.NewsFormDTO;
+import com.hubses.festival.repository.NewsRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import java.util.Optional;
+
+@Service
+public class NewsService {
+
+    @Autowired
+    private NewsRepository newsRepository;
+
+    public Optional<Iterable<News>> getAllNews() {
+        return Optional.ofNullable(newsRepository.findAll());
+    }
+
+    public Optional<News> getNewsById(long id) {
+        return Optional.ofNullable(newsRepository.findOne(id));
+    }
+
+    public Optional<News> deleteNewsById(long id) {
+        return newsRepository.findOneById(id).map(news -> {
```

Comment:
Why not findOneById(id).forEach(newsRepository::delete)? And what is expected this method to return? You can use same convention like in Java collections - return boolean which indicates whether it was a change (successful delete or not)